test/cases/openshift_virtualization.sh

```diff
@@ -0,0 +1,300 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source /usr/libexec/osbuild-composer-test/set-env-variables.sh
+source /usr/libexec/tests/osbuild-composer/shared_lib.sh
+
+BRANCH_NAME="${CI_COMMIT_BRANCH:-local}"
+BUILD_ID="${CI_JOB_ID:-$(uuidgen)}"
+
+# Provision the software under test.
+/usr/libexec/osbuild-composer-test/provision.sh none
+
+TEMPDIR=$(mktemp -d)
+function cleanup() {
+    greenprint "== Script execution stopped or finished - Cleaning up =="
+    # TODO: this needs to be added to cloud-cleaner as well
+    $OC_CLI delete vm "$VM_NAME"
+    $OC_CLI delete pvc "$PVC_NAME"
```

Comment:
This may result in "unbound" variable errors for `VM_NAME` and `PVC_NAME`, since this function can be executed at any time and these are set only after the compose succeeds.